examples/pytorch/graphsage/experimental/README.md

```diff
@@ -19,6 +19,11 @@ the number of nodes, the number of edges and the number of labelled nodes.
 python3 partition_graph.py --dataset ogb-product --num_parts 4 --balance_train --balance_edges
 ```
 
+```bash
+# partitoin for unsupervised learning
+python3 partition_graph.py --dataset ogb-product --num_parts 4 --balance_train --balance_edges --num_hops 2
```

Comment:
why 2 hops?

Reply:
I just copy the config from train_sampling_unsupervised.
It is just a hyperparameter

Reply:
but this is partition_graph.py. i don't think train_sampling_unsupervised invokes this script.

Reply:
removed